setup.py

```diff
@@ -124,12 +158,16 @@ def run(self):
     ],
     keywords="slack slack-web slack-rtm chat chatbots bots chatops",
     packages=find_packages(
-        exclude=["docs", "docs-src", "tests", "tests.*", "tutorial"]
+        exclude=["docs", "docs-src", "integration_tests", "tests", "tests.*", "tutorial"]
```

Comment:
the new director won't be included in the package.

<details>
<summary>`python3 setup.py sdist bdist_wheel --universal` outputs</summary>

```
$ python3 setup.py sdist bdist_wheel --universal
running sdist
running egg_info
writing slackclient.egg-info/PKG-INFO
writing dependency_links to slackclient.egg-info/dependency_links.txt
writing requirements to slackclient.egg-info/requires.txt
writing top-level names to slackclient.egg-info/top_level.txt
reading manifest file 'slackclient.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'slackclient.egg-info/SOURCES.txt'
running check
creating slackclient-2.6.0
creating slackclient-2.6.0/slack
creating slackclient-2.6.0/slack/rtm
creating slackclient-2.6.0/slack/web
creating slackclient-2.6.0/slack/web/classes
creating slackclient-2.6.0/slackclient.egg-info
copying files to slackclient-2.6.0...
copying LICENSE -> slackclient-2.6.0
copying MANIFEST.in -> slackclient-2.6.0
copying README.md -> slackclient-2.6.0
copying setup.cfg -> slackclient-2.6.0
copying setup.py -> slackclient-2.6.0
copying slack/__init__.py -> slackclient-2.6.0/slack
copying slack/errors.py -> slackclient-2.6.0/slack
copying slack/py.typed -> slackclient-2.6.0/slack
copying slack/version.py -> slackclient-2.6.0/slack
copying slack/rtm/__init__.py -> slackclient-2.6.0/slack/rtm
copying slack/rtm/client.py -> slackclient-2.6.0/slack/rtm
copying slack/web/__init__.py -> slackclient-2.6.0/slack/web
copying slack/web/base_client.py -> slackclient-2.6.0/slack/web
copying slack/web/client.py -> slackclient-2.6.0/slack/web
copying slack/web/slack_response.py -> slackclient-2.6.0/slack/web
copying slack/web/classes/__init__.py -> slackclient-2.6.0/slack/web/classes
copying slack/web/classes/actions.py -> slackclient-2.6.0/slack/web/classes
copying slack/web/classes/attachments.py -> slackclient-2.6.0/slack/web/classes
copying slack/web/classes/blocks.py -> slackclient-2.6.0/slack/web/classes
copying slack/web/classes/dialog_elements.py -> slackclient-2.6.0/slack/web/classes
copying slack/web/classes/dialogs.py -> slackclient-2.6.0/slack/web/classes
copying slack/web/classes/elements.py -> slackclient-2.6.0/slack/web/classes
copying slack/web/classes/interactions.py -> slackclient-2.6.0/slack/web/classes
copying slack/web/classes/messages.py -> slackclient-2.6.0/slack/web/classes
copying slack/web/classes/objects.py -> slackclient-2.6.0/slack/web/classes
copying slackclient.egg-info/PKG-INFO -> slackclient-2.6.0/slackclient.egg-info
copying slackclient.egg-info/SOURCES.txt -> slackclient-2.6.0/slackclient.egg-info
copying slackclient.egg-info/dependency_links.txt -> slackclient-2.6.0/slackclient.egg-info
copying slackclient.egg-info/requires.txt -> slackclient-2.6.0/slackclient.egg-info
copying slackclient.egg-info/top_level.txt -> slackclient-2.6.0/slackclient.egg-info
Writing slackclient-2.6.0/setup.cfg
Creating tar archive
removing 'slackclient-2.6.0' (and everything under it)
running bdist_wheel
running build
running build_py
installing to build/bdist.macosx-10.15-x86_64/wheel
running install
running install_lib
creating build/bdist.macosx-10.15-x86_64/wheel
creating build/bdist.macosx-10.15-x86_64/wheel/slack
copying build/lib/slack/version.py -> build/bdist.macosx-10.15-x86_64/wheel/slack
creating build/bdist.macosx-10.15-x86_64/wheel/slack/web
copying build/lib/slack/web/slack_response.py -> build/bdist.macosx-10.15-x86_64/wheel/slack/web
creating build/bdist.macosx-10.15-x86_64/wheel/slack/web/classes
copying build/lib/slack/web/classes/elements.py -> build/bdist.macosx-10.15-x86_64/wheel/slack/web/classes
copying build/lib/slack/web/classes/actions.py -> build/bdist.macosx-10.15-x86_64/wheel/slack/web/classes
copying build/lib/slack/web/classes/__init__.py -> build/bdist.macosx-10.15-x86_64/wheel/slack/web/classes
copying build/lib/slack/web/classes/dialog_elements.py -> build/bdist.macosx-10.15-x86_64/wheel/slack/web/classes
copying build/lib/slack/web/classes/attachments.py -> build/bdist.macosx-10.15-x86_64/wheel/slack/web/classes
copying build/lib/slack/web/classes/messages.py -> build/bdist.macosx-10.15-x86_64/wheel/slack/web/classes
copying build/lib/slack/web/classes/interactions.py -> build/bdist.macosx-10.15-x86_64/wheel/slack/web/classes
copying build/lib/slack/web/classes/dialogs.py -> build/bdist.macosx-10.15-x86_64/wheel/slack/web/classes
copying build/lib/slack/web/classes/blocks.py -> build/bdist.macosx-10.15-x86_64/wheel/slack/web/classes
copying build/lib/slack/web/classes/objects.py -> build/bdist.macosx-10.15-x86_64/wheel/slack/web/classes
copying build/lib/slack/web/base_client.py -> build/bdist.macosx-10.15-x86_64/wheel/slack/web
copying build/lib/slack/web/client.py -> build/bdist.macosx-10.15-x86_64/wheel/slack/web
copying build/lib/slack/web/__init__.py -> build/bdist.macosx-10.15-x86_64/wheel/slack/web
copying build/lib/slack/__init__.py -> build/bdist.macosx-10.15-x86_64/wheel/slack
creating build/bdist.macosx-10.15-x86_64/wheel/slack/rtm
copying build/lib/slack/rtm/client.py -> build/bdist.macosx-10.15-x86_64/wheel/slack/rtm
copying build/lib/slack/rtm/__init__.py -> build/bdist.macosx-10.15-x86_64/wheel/slack/rtm
copying build/lib/slack/py.typed -> build/bdist.macosx-10.15-x86_64/wheel/slack
copying build/lib/slack/errors.py -> build/bdist.macosx-10.15-x86_64/wheel/slack
running install_egg_info
Copying slackclient.egg-info to build/bdist.macosx-10.15-x86_64/wheel/slackclient-2.6.0-py3.8.egg-info
running install_scripts
adding license file "LICENSE" (matched pattern "LICEN[CS]E*")
creating build/bdist.macosx-10.15-x86_64/wheel/slackclient-2.6.0.dist-info/WHEEL
creating 'dist/slackclient-2.6.0-py2.py3-none-any.whl' and adding 'build/bdist.macosx-10.15-x86_64/wheel' to it
adding 'slack/__init__.py'
adding 'slack/errors.py'
adding 'slack/py.typed'
adding 'slack/version.py'
adding 'slack/rtm/__init__.py'
adding 'slack/rtm/client.py'
adding 'slack/web/__init__.py'
adding 'slack/web/base_client.py'
adding 'slack/web/client.py'
adding 'slack/web/slack_response.py'
adding 'slack/web/classes/__init__.py'
adding 'slack/web/classes/actions.py'
adding 'slack/web/classes/attachments.py'
adding 'slack/web/classes/blocks.py'
adding 'slack/web/classes/dialog_elements.py'
adding 'slack/web/classes/dialogs.py'
adding 'slack/web/classes/elements.py'
adding 'slack/web/classes/interactions.py'
adding 'slack/web/classes/messages.py'
adding 'slack/web/classes/objects.py'
adding 'slackclient-2.6.0.dist-info/LICENSE'
adding 'slackclient-2.6.0.dist-info/METADATA'
adding 'slackclient-2.6.0.dist-info/WHEEL'
adding 'slackclient-2.6.0.dist-info/top_level.txt'
adding 'slackclient-2.6.0.dist-info/RECORD'
removing build/bdist.macosx-10.15-x86_64/wheel
```

</details>